ivy/func_wrapper.py

```diff
@@ -777,7 +777,7 @@ def _infer_device(*args, device=None, **kwargs):
 
 def handle_out_argument(fn: Callable) -> Callable:
     handle_out_in_backend = hasattr(fn, "support_native_out")
-    handle_out_in_ivy = hasattr(fn, "mixed_function")
+    is_compositional_function = "ivy.functional.ivy" in fn.__module__
```

Comment:
it would be great if we had a smaller variable name here 😅

Reply:
Sure, done :sweat_smile: